T7Suite/AFRMap.cs

```diff
@@ -509,9 +510,10 @@ private byte[] LoadTargetAFRMapInBytes(int[] orginalmap, int size)
                     map.SetValue(b2, idx++);
                 }
             }
-            catch (Exception)
+            catch (Exception E)
             {
                 //   MessageBox.Show("Failed to load target AFR map: " + E.Message);
```

Comment:
Maybe remove the MessageBox comment, take the error sting and add it to the logger.Debug() together with E.